tools/ci_build/github/windows/bundle_dlls_gpu.bat

```diff
@@ -10,6 +10,11 @@ FOR /R %%i IN (*.nupkg) do (
        mkdir runtimes\linux-x64\native
        move onnxruntime-linux-x64\lib\libonnxruntime.so.1* runtimes\linux-x64\native\libonnxruntime.so
        move onnxruntime-linux-x64\lib\libonnxruntime_providers_* runtimes\linux-x64\native
+       mkdir runtimes\win-x64\native
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime_providers_tensorrt.dll runtimes\win-x64\native\onnxruntime_providers_tensorrt.dll
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime.dll runtimes\win-x64\native\onnxruntime.dll
```

Comment:
line 15-17 are not needed?

Reply:
They are needed. We must use onnxruntime.dll that is built with USE_TENSORRT, otherwise we will get this error:
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/session/provider_bridge_ort.cc#L1190

Reply:
Then does the TRT one built with USE_CUDA? If yes, why not get the cuda dlls from there too. If not, CUDA wouldn't work?

Reply:
Yes, TRT build also built with USE_CUDA.
The reason we don't use CUDA dlls from TRT build is that we need to handle the macro "USE_TENSORRT" properly.
That's why previously we discussed if we want to do so, we need to remove all the macro "USE_TENSORRT" that might affect CUDA code path. But for some reason we can't remove all of them, therefore we took another approach as I mentioned in the PR description, extract only extract TRT EP shared library and onnxruntime.dll only.

Reply:
After you added TRT there, the package actually gets 10% smaller... I'm looking why.